packages/primitives/popover/src/popover.utils.ts

```diff
@@ -98,11 +98,9 @@ export function getArrowPositionParams(
     arrowWidthAndHeight: { width: number; height: number },
     triggerWidthAndHeight: { width: number; height: number }
 ): RdxArrowPositionParams {
-    const posParams = {
+    const posParams: RdxArrowPositionParams = {
```

Comment:
Could you clarify the logic here?

There's an if condition earlier checking for `[RdxPopoverSide.Top, RdxPopoverSide.Bottom]`, but further on, we're only dealing with `left` & `top`.

Reply:
> Could you clarify the logic here?
> 
> There's an if condition earlier checking for `[RdxPopoverSide.Top, RdxPopoverSide.Bottom]`, but further on, we're only dealing with `left` & `top`.

@pimenovoleg yes, sure. For absolute positioninig we can use `left` or `right` for `x` positioning and `top` or `bottom` for `y` positioning. For some cases I used `left` and for the others `right` and the same for `top` and `bottom`. Now. for the sake of simplicity I use only `left` and `top` for absolute positioning. The result is always the same.